app/controllers/standards_imports_controller.rb

```diff
@@ -10,6 +10,9 @@ def create
 
     unless user_signed_in?
       @standards_import.courtesy_notification = :pending
+      @standards_import.imports.each do |import|
+        import.courtesy_notification = :pending
```

Comment:
I'd like to learn more about this step. Is this something we can set from `StandardsImport#files=`?

Reply:
I tried that first, but since we are not passing in `courtesy_notification` as a form field, we can't set it in `files=`, which gets called on line 9. Any other ideas? We could pass it in as a form field, but that bugs me a bit since it isn't necessary to come in on the form.

Reply:
I came up with something uglier but deleted that.

Is this something we can set before_save? Spitballing here:

```ruby
class Imports < ApplicationRecord
  before_save :set_courtesy_notification

  def standard_courtesy_notification
    parent.standard_courtesy_notification
  end

  private

  def set_courtesy_notification
    self.courtesy_notification = standard_courtesy_notification
  end
end
```

```ruby
class StandardsImport < ApplicationRecord
  def standard_courtesy_notification
    courtesy_notification
  end
end
```

Then we can keep line 12 in the controller but drop 13-15.

Is that worse?

Reply:
Oh I like that way better. Thanks!

Reply:
Fixed in c86aa55c70d6fc48338b52ce107f9e1b856b9286